frame/plasm-rewards/src/lib.rs

```diff
@@ -0,0 +1,428 @@
+//! # Plasm Staking Module
+//!
+//! The Plasm staking module puts together the management and compensation payment logic of the ERA.
+//! The Plasm Rewards module calls the Dapps Staking and Validator.
+//! It also allocates rewards to each module according to the [Plasm Token Ecosystem inflation model](https://docs.plasmnet.io/PlasmNetwork/TokenEcosystem.html).
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::{Decode, Encode};
+use session::SessionManager;
+use sp_runtime::{
+    traits::{SaturatedConversion, Zero},
+    Perbill, RuntimeDebug,
+};
+use sp_std::{prelude::*, vec::Vec};
+pub use staking::Forcing;
+use support::{
+    decl_error, decl_event, decl_module, decl_storage,
+    traits::{Currency, Get, LockableCurrency, Time},
+    weights::SimpleDispatchInfo,
+    StorageMap, StorageValue,
+};
+use system::ensure_root;
+
+pub mod inflation;
+#[cfg(test)]
+mod mock;
+pub mod traits;
+use traits::*;
+#[cfg(test)]
+mod tests;
+
+pub use sp_staking::SessionIndex;
+
+pub type EraIndex = u32;
+pub type BalanceOf<T> =
+    <<T as Trait>::Currency as Currency<<T as system::Trait>::AccountId>>::Balance;
+pub type MomentOf<T> = <<T as Trait>::Time as Time>::Moment;
+
+// A value placed in storage that represents the current version of the Staking storage.
+// This value is used by the `on_runtime_upgrade` logic to determine whether we run
+// storage migration logic. This should match directly with the semantic versions of the Rust crate.
+#[derive(Encode, Decode, Clone, Copy, PartialEq, Eq, RuntimeDebug)]
+enum Releases {
+    V1_0_0Ancient,
```

Comment:
May be is better to use semantic versioning scheme? I mean `Ancient` tag is doesn't make sense here.

Reply:
Do you mean using the Plasm version?

Reply:
I mean using [this](https://semver.org/) convention without additional labels like `Ancient`.